server.go

```diff
@@ -407,6 +412,9 @@ func (s *Server) Stop() error {
 		return err
 	}
 
+	// Reset start flag in case of future restart.
+	atomic.StoreInt32(&s.started, 0)
```

Comment:
Is this for the itest? Would assume the struct would just be re-created anew?

Reply:
You're right, I've dropped these changes in the latest commits.